swan-lake/integration-tools/graphql-tool.md

```diff
@@ -8,15 +8,57 @@ active: graphql-tool
 intro: The Ballerina GraphQL tool makes it easy to start the development of a GraphQL client. It generates the client skeletons in Ballerina for a given GraphQL schema and a GraphQL document. In addition, the GraphQL tool supports generating the GraphQL schema for a given Ballerina GraphQL service and writing it to a file in GraphQL schema definition language (SDL).
 --- 
 
+## Usage
+
 The Ballerina GraphQL tooling support provides the following capabilities.
 
-1. Generating a Ballerina client from a given GraphQL config file configured with a GraphQL schema, which is specified by the GraphQL schema definition language and a GraphQL document/documents.
+1. Generate a Ballerina client from a given GraphQL config file configured with a GraphQL schema, which is specified by the GraphQL schema definition language and a GraphQL document/documents.
 
-2. Generating multiple Ballerina modules from a given GraphQL config file configured with multiple GraphQL projects.
+2. Generate multiple Ballerina modules from a given GraphQL config file configured with multiple GraphQL projects.
 
 3. Generate the GraphQL schema for a given Ballerina GraphQL service(s) and write the schema(s) to a file/files using the GraphQL schema definition language.
```

Comment:
If this is targeting Update 8, we should include the Ballerina service code stub generation support from GraphQL schema. (I think we already merged the relevant changes in this doc @DimuthuMadushan?)

Reply:
Yes. Related PR: https://github.com/ballerina-platform/ballerina-dev-website/pull/6959/files

Reply:
These changes will go live before Update 8. Hence, the content for Update 8 will be added later.